Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Location/LocationDrawings.tsx

```diff
@@ -155,128 +67,45 @@ const LocationDrawingsWindow = (props: { Location: OpenXDA.Types.Location }) =>
-                        > Link
-                        </ReactTable.Column>
-                        <ReactTable.Column<SystemCenter.Types.LocationDrawing>
-                            Key={'Description'}
-                            AllowSort={true}
-                            Field={'Description'}
-                            HeaderStyle={{ width: 'auto' }}
-                            RowStyle={{ width: 'auto' }}
-                        > Description
-                        </ReactTable.Column>
-                        <ReactTable.Column<SystemCenter.Types.LocationDrawing>
-                            Key={'Number'}
-                            AllowSort={true}
-                            Field={'Number'}
-                            HeaderStyle={{ width: 'auto' }}
-                            RowStyle={{ width: 'auto' }}
-                        > Number
-                        </ReactTable.Column>
-                        <ReactTable.Column<SystemCenter.Types.LocationDrawing>
-                            Key={'Category'}
-                            AllowSort={true}
-                            Field={'Category'}
-                            HeaderStyle={{ width: 'auto' }}
-                            RowStyle={{ width: 'auto' }}
-                        > Category
-                        </ReactTable.Column>
-                        <ReactTable.Column<SystemCenter.Types.LocationDrawing>
-                            Key={'EditDelete'}
-                            AllowSort={false}
-                            HeaderStyle={{ width: 'auto' }}
-                            RowStyle={{ width: 'auto' }}
-                            Content={({ item }) =>
-                                <span>
-                                    <button title='Edit Link' className={"btn" + (!hasPermissions() ? ' disabled' : '')} data-toggle={"modal" + (!hasPermissions() ? ' disabled' : '')} data-target="#exampleModal" onClick={(e) => {setRecord(item) }}>{Pencil}</button>
-                                    <button title='Delete Link' className={"btn" + (!hasPermissions() ? ' disabled' : '')} onClick={(e) => { if (hasPermissions()) handleDelete(item); }}>{TrashCan}</button>
-                                </span>
-                            }
-                        > <p></p>
-                        </ReactTable.Column>
-                    </ReactTable.Table>
-                    <LoadingScreen Show={pageState == 'loading'} />
-                    <ServerErrorIcon Show={pageState == 'error'} Size={40} Label={'A Server Error Occurred. Please Reload the Application.'} />
-                    <div className="row">
-                        <div className="col">
-                            <Paging Current={page + 1} Total={pageInfo.NumberOfPages} SetPage={(p) => setPage(p - 1)} />
-                        </div>
-                    </div>
-                </div>
+                <LocationDrawingsTable
```

Comment:
Why did we break this out into its own file? Is it used somewhere else?

Reply:
Yes in two places.